install_ads.bat

```diff
@@ -171,17 +171,11 @@ ads_conda\Scripts\conda create -y -p ads_conda\envs\venv_ads python=3.11 || goto
 CALL ads_conda\Scripts\activate.bat ads_conda\envs\venv_ads || goto error
 echo Virtual environment created and activated successfully!
 
-rem Install ADS and its requirements
-if exist requirements-freeze.txt (
-  set requirements_file=requirements-freeze.txt || goto error
-) else (
-  set requirements_file=requirements.txt || goto error
-)
 echo:
-echo ### Installing ADS and its dependencies from %requirements_file%...
+echo ### Installing ADS and its dependencies
 rem Skip pip==21.2 to avoid dependency resolver issue (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3593)
 ads_conda\envs\venv_ads\python -m pip install -U "pip^!=21.2.*" || goto error
-ads_conda\envs\venv_ads\Scripts\pip install -r %requirements_file% || goto error
+ads_conda\envs\venv_ads\Scripts\pip install -e . || goto error
 ads_conda\envs\venv_ads\Scripts\pip install -e . plugins\ --use-pep517 || goto error

```

Comment:
This step is duplicated, I think?

Reply:
Good catch - I didn't notice the `.`. in the plugins install line below, I thought that line was just for the plugin. I'll remove this line (178)

Reply:
Resolved in 3762ae3